srcpkgs/glslViewer/template

```diff
@@ -1,31 +1,28 @@
 # Template file for 'glslViewer'
 pkgname=glslViewer
-version=1.6.8
+version=3.0.7
 revision=1
-build_style=gnu-makefile
-make_use_env=yes
+_vera_gitrev="91c48874900909ebd58440359657948aed906f50"
```

Comment:
Where did you get this revision from? https://github.com/patriciogonzalezvivo/glslViewer/tree/3.0.7/deps seems to suggest it's wrong.

Reply:
It seems I had gotten this revision from the master branch, not the tagged one.

I am guilty on my part.